screen.py

```diff
@@ -0,0 +1,306 @@
+import pygame
+import colors
+from teams import Team, get_score_dif
+import os
+
+pygame.init()
+
+SCREEN_WIDTH = 480
+SCREEN_HEIGHT = 680
+MIDDLE_HORIZONTAL = SCREEN_WIDTH / 2
+RECT_WIDTH = 200
+RECT_HEIGHT = 100
+
+SMALL_RECT_WIDTH = 60
+SMALL_RECT_HEIGHT = SCREEN_HEIGHT/10
+
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+squares = []
+PICTURES_PATH = 'pictures'
+
+BOARD_LINE = 8
+HEIGHT_OF_SCOREBOARD = 200
+SCORE_BOARD = pygame.Surface((screen.get_width(), HEIGHT_OF_SCOREBOARD))
+REGULAR_FONT = pygame.font.SysFont('comicsansms', 30)
+LARGE_FONT = pygame.font.Font('freesansbold.ttf', 40)
+
+
+class ExitGame(Exception):
+    pass
+
+
+class Square:
+    WIDTH = 60
+    HEIGHT = 60
+
+    def __init__(self, x, y, color, tur, line):
+        self.rect = pygame.Rect(x, y, self.WIDTH, self.HEIGHT)
+        self.color = color
+        self.original_color = color
+        self.tur_cord = tur
+        self.line_cord = line
+        self.current_piece = None
+
+    def draw(self):
+        pygame.draw.rect(screen, self.color, self.rect)
+
+    def coloring_square_by_original_color(self):
+        if self.color == self.original_color:
+            if self.original_color == colors.DARK_BROWN:
+                self.color = colors.DARK_RED
+            else:
+                self.color = colors.LIGHT_RED
+        else:
+            self.color = self.original_color
+
+    def __str__(self):
+        return f'(line: {self.line_cord}, tur: {self.tur_cord})'
+
+
+def is_move_to_square_valid(tur, line, team):
+    """
+    Check if square is in board bounds and not taken by teammate piece.
```

Comment:
Just in this function you have a docstring, delete it or add docstring to all other functions.

Reply:
:1234: